testsuite/modules.70-maint/410-protected_envvars.exp

```diff
@@ -0,0 +1,55 @@
+##############################################################################
+#   Modules Revision 3.0
+#   Providing a flexible user environment
+#
+#   File:		modules.70-maint/%M%
+#   Revision:		%I%
+#   First Edition:	2022/06/01
+#   Last Mod.:		%U%, %G%
+#
+#   Authors:		Adrien Cotte, adrien@cotte.com
+#
+#   Description:	Testuite testsequence
+#   Command:
+#   Modulefiles:
+#   Sub-Command:
+#
+#   Comment:	%C{
+#           Test 'protected_envvars' config
+#		}C%
+#
+##############################################################################
+
+set mp $modpath.3
+set mpre $modpathre.3
+setenv_path_var MODULEPATH $mp
+
+set mod protect/1.0
+set modname protect
+set modfile $mp/$mod
+set modfilere $mpre/$mod
+
+#
+#  Tests
+#
+
+set ans [list]
+lappend ans [list set _LMFILES_ $modfile]
+lappend ans [list set LOADEDMODULES $mod]
+lappend ans [list ERR]
```

Comment:
The introduced protected variable mechanism emits a warning message when trying to modify a variable set as protected. This mechanism does not lead to an error, so the tests do not output an error shell code. If you remove this `lappend ans [list ERR]` line, the tests should pass.

Reply:
Damn it, so obvious...
Many thanks, again.